### DISCUSSION TITLE
How to find out the ReadCount post Downsampling??

### DISCUSSION THREAD

**Discussion by meg-gits-it** [milaboratory/mixcr#1248] [Q&A] [answered]
How to generate a file containing the ReadCount field, after generating the .json files and posanalysis metrics file from downsampling, to find out the number of reads in the downsampled data?

**Comment by meg-gits-it** [comment]
How to generate a clones.tsv file from .json file

**Comment by mizraelson** [comment, marked as answer]
Hi,

Could you please share the commands you've used and clarify the specific results you're aiming for?

To briefly guide you, if you run the command as shown below:

```
mixcr postanalysis individual \
    --metadata metadata.tsv \
    --default-downsampling count-reads-auto \
    --default-weight-function read \
    --only-productive \
    --chains IGH \
    --tables pa/pa.i.tsv \
    --preproc-tables pa/preproc.i.tsv \
    results/*.clns \
    pa/individual.json.gz
```

The `preproc.i.tsv` tables will contain information regarding the number of clones and reads after every filtering step (for entire sample), including downsampling. You can reference an example [here](https://mixcr.com/mixcr/guides/generic-multiplex-bcr/#individual-postanalysis).

Note that you cannot extract clonotype tables directly from a .json file. However, you can utilize the original `.clns` files to produce downsampled `.clns` files using [`mixcr downsample`](https://mixcr.com/mixcr/reference/mixcr-downsample/). Subsequently, you can execute `mixcr exportClones` to obtain downsampled clones in tsv tables where you will see a `ReadCount` value for every clone.

I hope this assists you. Please let me know if you have further questions.
